### PR TITLE
New testfile structure

### PR DESCRIPTION
**Pull request by jbathmann**
I removed the old testfiles and moved everything to -> Benchmarks

From the SimpleSetups_I, nly the version with all simple survived.
For the other Benchmarks, the module tests are included in their short version.
In- and exclusion of benchmarks can be done in the BenchmarkList.xml file

### CODE REVIEW

**Review by mcwimm — COMMENTED** [review]
First, 'test_Benchmarks_CI.py' was not working (Error: AttributeError: 'DataFrame' object has no attribute 'compare'). Solution: update pandas (==1.1.5).
Then it worked. 

The output confuses me a little. 
'Ran 1 test in 6.695s' - for me, this means only 1 test was done but in ran all tests listed in BenchmarkList.xml.

to be continued....

File: Benchmarks/test_Benchmarks_CI.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import sys
 from os import path
 import os
```
[the remark this answers is not in the document]
Comment (jbathmann) [reply Benchmarks/test_Benchmarks_CI.py:3]: removed

**Comment by jbathmann** [comment]
> The output confuses me a little.
> 'Ran 1 test in 6.695s' - for me, this means only 1 test was done but in ran all tests listed in BenchmarkList.xml.

For pytest, only on test was ran. However, the test _(test_benchmarks)_ in _(test_Benchmarks_CI.py)_ consists of N subtests (line 44ff.). The number of subtests depends on the number of tests listed in BenchmarkList.xml

> I stopped the review because files online are different from checked out branch. @jbathmann can you compare it with your local status?

I checked and it should be fine now?

**Review by mcwimm — COMMENTED** [review]
I know it's annoying, but could you please add a few comments in the script 'test_Benchmarks_CI'  :)

Apart from that everything works.

**Comment by jbathmann** [comment]
Updated. @mcwimm please approve PR, if you think it is ready to merge.

**Review by mcwimm — APPROVED** [review]
👍

**Pull request merged**